staffeli_nt/download.py

```diff
@@ -201,21 +203,22 @@ def ta_file(args):
             with open(path, 'wb') as bf:
                 bf.write(data)
 
-            # unzip attachments
-            if attachment['mime_class'] == 'zip':
-                unpacked = os.path.join(base, 'unpacked')
-                os.mkdir(unpacked)
-                try:
-                    with zipfile.ZipFile(path, 'r') as zip_ref:
-                        try:
-                            zip_ref.extractall(unpacked)
-                            # Run through onlineTA
-                            if template.onlineTA is not None:
-                                run_onlineTA(base, unpacked, template.onlineTA)
-                        except NotADirectoryError:
-                            print(f"Attempted to unzip into a non-directory: {name}")
-                except BadZipFile:
-                    print(f"Attached archive not a zip-file: {name}")
+            if no_unzip:
+                # unzip attachments
```

Comment:
Either there is a logical error, or the variable `no_unzip` has a confusing name. To me the code read as "If the option `--no-unzip` then unzip attachments"